### DISCUSSION TITLE
Running lower “resolution” options

### DISCUSSION THREAD

**Discussion by mikeobr** [komoot/photon#831] [Q&A] [answered]
Hello, geocoding noob here. If I want to save on resources and compute and do not need address level data, but more just suburb, city level and up, is that supported and how does one go about that?

**Comment by lonvia** [comment, marked as answer]
You would need to set up your own Nominatim instance with the limited data and then do your own Photon export from that. There is currently no facility to filter from the extracts you provide.

The Nominatim import can be run with the [admin style](https://nominatim.org/release-docs/latest/admin/Import/#filtering-imported-data), which takes only a couple of hours to finish and pretty much gives you what you want. Then create the Photon database as [described in the README](https://github.com/komoot/photon?tab=readme-ov-file#customized-search-data).
